codebase/superdarn/src.idl/app/makefit.1.5/calcphires.pro

```diff
@@ -1,31 +1,26 @@
 ; calcphires.pro
 ; ==============
 ; Author: R.J.Barnes - Based on C code by K.Baker
-; 
-; LICENSE AND DISCLAIMER
-; 
+;
 ; Copyright (c) 2012 The Johns Hopkins University/Applied Physics Laboratory
-; 
-; This file is part of the Radar Software Toolkit (RST).
-; 
-; RST is free software: you can redistribute it and/or modify
-; it under the terms of the GNU Lesser General Public License as published by
-; the Free Software Foundation, either version 3 of the License, or
-; any later version.
-; 
-; RST is distributed in the hope that it will be useful,
-; but WITHOUT ANY WARRANTY; without even the implied warranty of
-; MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
-; GNU Lesser General Public License for more details.
-; 
-; You should have received a copy of the GNU Lesser General Public License
-; along with RST.  If not, see <http://www.gnu.org/licenses/>.
-; 
-; 
-; 
 ;
+;RST is free software: you can redistribute it and/or modify
+;it under the terms of the GNU General Public License as published by
+;the Free Software Foundation, either version 3 of the License, or
+;(at your option) any later version.
+;
+;This program is distributed in the hope that it will be useful,
+;but WITHOUT ANY WARRANTY; without even the implied warranty of
+;MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+;GNU General Public License for more details.
 ;
-; PublicFunctions
+;You should have received a copy of the GNU General Public License
+;along with this program. If not, see <https://www.gnu.org/licenses/>.
+;
+;Modifications:
+; 
+; RST is free software: you can redistribute it and/or modify
+; it under the terms of the GNU General Public License as published by
 ; ---------------
```

Comment:
It does not seem like the license/copyright header information was correctly imported into the example IDL makefit app files.

Reply:
Looks like the last 2 lines here should be removed.  I should be able to address these in the next few hours.

Reply:
These files have been resolved with ce449d1.